pkg/cmd/serviceaccount/list/list.go

```diff
@@ -71,6 +75,10 @@ func NewListCommand(f *factory.Factory) *cobra.Command {
 
 	_ = cmd.Flags().MarkDeprecated("enable-auth-v2", opts.localizer.MustLocalize("serviceAccount.common.flag.deprecated.enableAuthV2"))
 
+	cmd.Flags().Int32VarP(&opts.page, "page", "", int32(cmdutil.ConvertPageValueToInt32(build.DefaultPageNumber)), opts.localizer.MustLocalize("serviceAccount.list.flag.page.description"))
```

Comment:
Are we adding a `--page` flag? The flag doesn't work for me:

```shell
╰$ ./rhoas service-account --page 2
❌ Unknown flag: --page. Run the command in verbose mode using the -v flag to see more information
```

Another feature that might be nice is letting the user know how many service accounts there are? I know we output the number of service accounts displayed using -> `(100)`, but as a user I'd like to know if there are more, and if so how to get them. I'm unsure if this is difficult?

This would mean that the `-h` has to be updated too.

Reply:
Hi @dimakis 
The command you'd like to run is `rhoas service-account list --page 2`.

We have thought of improving the overall experience with pagination in CLI by providing additional information. Though it doesn't seem to be possible for service-accounts as of now.

The response we get is an array, so there is no way to know the total no. of service accounts.